app/2.0/docs/tools/polymer-json.md

```diff
@@ -146,7 +146,14 @@ required.
 * `swPrecacheConfig`: An optional configuration file for the generated service worker.
 * `insertPrefetchLinks`: If `true`, insert prefetch link elements into your fragments so that all
 dependencies are prefetched immediately.
-* `bundle`: If `true`, bundle your application.
+* `bundle`: May be boolean, or an object bag of settings matching specified [`polymer-bundler`](https://github.com/Polymer/polymer-bundler) constructor options.
+  * If `true`, bundle your application.
+  * If an object is provided with further configuration, the options are:
+    * `excludes`: Optional, `String[]`. Excludes the listed paths of files or folders from inlining.
+    * `inlineCss`: Optional, `Boolean`. Specifies whether to inline external CSS file contents into `<style>` tags.
+    * `inlineScripts`: Optional, `Boolean`. Specifies whether to inline external JavaScript file contents into `<script>` tags.
+    * `rewriteUrlsInTemplates`: Optional, `Boolean`. Specifies whether to rewrite element attributes inside of templates when inlining HTML.
```

Comment:
As an aside, this feature was how Polymer 1.x did things and some people with Polymer 1.x projects and elements may need this.  To the extent you think that provides value, maybe a small mention that this emulates the Polymer 1.x behavior.